build.sbt

```diff
@@ -139,4 +176,90 @@ lazy val java = project.in(file("faunadb-java"))
       "org.yaml" % "snakeyaml" % "1.14" % "test",
       "com.novocode" % "junit-interface" % "0.11" % "test",
       "org.hamcrest" % "hamcrest-library" % "1.3" % "test",
-      "junit" % "junit" % "4.12" % "test"))
+      "junit" % "junit" % "4.12" % "test"
+    )
+  )
+
+lazy val javaAndroid = project.in(file("faunadb-android"))
+  .dependsOn(javaDsl)
+  .settings(publishSettings: _*)
+  .settings(androidBuild: _*)
+  .settings(
+    name := "faunadb-android",
+    crossPaths := false,
+    autoScalaLibrary := false,
+    javacOptions ++= Seq("-source", "1.7", "-target", "1.7"),
+    testOptions += Tests.Argument(TestFrameworks.JUnit, "+q", "-v"),
+    apiURL := Some(url("http://faunadb.github.io/faunadb-jvm/faunadb-java/api/")),
+
+    (javacOptions in doc) := Seq("-source", "1.7",
+      "-link", "http://docs.oracle.com/javase/7/docs/api/",
+      "-link", s"http://google.github.io/guava/releases/$guavaVersion/api/docs/",
+      "-link", s"http://fasterxml.github.io/jackson-databind/javadoc/$jacksonDocVersion/",
+      "-link", s"http://faunadb.github.io/faunadb-jvm/$driverVersion/faunadb-common/api/",
+      "-link", s"http://dropwizard.github.io/metrics/$metricsVersion/apidocs/",
+      "-link", "http://www.joda.org/joda-time/apidocs/"
+    ),
+
+    testOptions += Tests.Argument(TestFrameworks.JUnit, "-q"),
+
+    libraryDependencies ++= Seq(
+      "com.squareup.okhttp3" % "okhttp" % "3.4.1",
+      "com.google.guava" % "guava" % "19.0",
+      "com.google.code.findbugs" % "jsr305" % "2.0.1",
+      "com.novocode" % "junit-interface" % "0.11" % "test",
+      "org.hamcrest" % "hamcrest-library" % "1.3" % "test",
+      "junit" % "junit" % "4.12" % "test"
+    ),
+
+    platformTarget in Android := "android-16",
+    buildToolsVersion in Android := Some("24.0.2"),
+    showSdkProgress in Android := true,
+    useProguard := true,
+    useProguardInDebug := true,
+
+    publishArtifact in Test := false,
+    publishArtifact in Compile := true,
+    publishArtifact in (Compile, packageBin) := true,
+    publishArtifact in (Compile, packageDoc) := true,
+    publishArtifact in (Compile, packageSrc) := true,
+
+    proguardOptions in Android ++= Seq(
```

Comment:
When the error will be reported? When we compile the jar or when some customer tries to use the jar?